server/api/v1/example.go

```diff
@@ -0,0 +1,53 @@
+package v1
+
+import (
+	"github.com/gin-gonic/gin"
+)
+
+// we could make one of these per service or one for all of v1 depending on coupling points
```

Comment:
Im not sure how exactly we want to structure this but i think we will know more when we get rolling more.